vpr/src/base/atom_netlist.cpp

```diff
@@ -174,7 +175,8 @@ AtomPortId AtomNetlist::create_port(const AtomBlockId blk_id, const t_model_port
     if (!port_id) {
         port_id = Netlist::create_port(blk_id, model_port->name, model_port->size, type);
 
-        port_models_.push_back(model_port);
+        // Use insert instead of push_back
```

Comment:
Same comment here. Please use the exact same comment that you use above. It may seem silly to duplicate comments, but it makes it clear that we are using the same "technique" everywhere.